unit-test/background/https.es6.js

```diff
@@ -4,6 +4,7 @@ const httpsStorage = require('../../shared/js/background/storage/https.es6')
 const httpsBloom = require('./../data/httpsBloom.json')
 const httpsWhitelist = require('./../data/httpsWhitelist.json')
 const load = require('./../helpers/https.es6')
+const newTab = require('./../data/newTab.json')
```

Comment:
I'm not fully sure why, but tests error out when I try to include the tab class. I get the error message `Uncaught ReferenceError: chrome is not defined`.

The newTab.json object I'm using here is just a fake tab object.

Reply:
It's because of the call to `browserWrapper.setBadgeIcon()`, which assumes an extension context. You'd need to stub that out. See the Site unit tests for an example: https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/7c1bf9472ab5d55114586ea2c346afa8364de8a8/unit-test/background/classes/site.es6.js#L7-L9

Reply:
Fixed in #337, see https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/337/files#r232644157.